clams/develop/templates/app/app.py.template

```diff
@@ -39,6 +39,11 @@ class $APP_CLASS_NAME(ClamsApp):
         # see https://sdk.clams.ai/autodoc/clams.app.html#clams.app.ClamsApp._annotate
         raise NotImplementedError
 
+def get_app():
+    # singleton constructor for app
+    # return $APP_CLASS_NAME(init, from, global, params)
```

Comment:
Guess we, as SDK developers, can give more detailed documentation on how an app developer can or should use global variables in this constructor function.